.github/workflows/VersionCalPRComment.yml

```diff
@@ -0,0 +1,131 @@
+name: Comment PR new module release when pr committed and labeled/unlabeled
+
+on: 
+  workflow_dispatch:
+  pull_request:
```

Comment:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflows-in-forked-repositories
With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository. 
The GITHUB_TOKEN has read-only permissions in pull requests from forked repositories. 
In my impression, this will cause the workflow to not take effect in most cases, because most users use the forked repo, please test carefully before and after going online.